book/content/version_control/version_control.md

```diff
@@ -723,7 +723,7 @@ This also works for using commit messages to close issues too.
 
 ## What to learn next
 
-Look into best practice for writing good quality code (for example, good naming conventions, informative comments, modular code structure).
+Look into [best practice for writing good quality code](../code_quality/code_quality) (for example, good naming conventions, informative comments, modular code structure).
```

Comment:
```suggestion
Look into [best practice for writing good quality code](../code_quality/code_quality) (eg. good naming conventions, informative comments, modular code structure).
```

Reply:
The style guide and CI don't like Latin abbreviations :(